components/support/jwcrypto/src/direct.rs

```diff
@@ -0,0 +1,159 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// Support for "Direct Encryption with a Shared Symmetric Key"
```

Comment:
Similarly, it's probably worth mentioning and linking to [RFC7518 Section 4.5](https://tools.ietf.org/html/rfc7518#section-4.5) here for easy context for the reader.